code/modules/projectiles/projectile/special.dm

```diff
@@ -69,6 +69,7 @@
 	desc = "Boom"
 	icon_state = "missile"
 	damage = 30
+	ricochets_max = 0 //it's a MISSILE
```

Comment:
I'm pretty sure that's for reflector vests, as the ricochet code doesn't check it at all.
Still, I'll take a look.